src/applications/common/schemaform/save-in-progress/RoutedSavableReviewPage.jsx

```diff
@@ -103,18 +118,35 @@ class RoutedSavableReviewPage extends React.Component {
       user
     } = this.props;
 
+    // const submitController = (
+    // );
+    //
+    // let submitContent = submitController;
+    //
+    // if (formConfig.downtime) {
+    //   submitContent = (
+    //   );
+    // }
+    //
+    const downtimeDependencies = get('downtime.dependencies', formConfig) || [];
+
     return (
       <div>
         <ReviewChapters
           formConfig={formConfig}
           formContext={formContext}
           pageList={pageList}
           onSetData={() => this.debouncedAutoSave()}/>
-        <SubmitController
-          formConfig={formConfig}
-          pageList={pageList}
-          path={path}
-          renderErrorMessage={this.renderErrorMessage}/>
+        <DowntimeNotification
```

Comment:
@rianfowler This is the simpler approach, just with the downside of always rendering DowntimeNotification.